languages/csharp/exercises/concept/string-formatting/.docs/introduction.md

```diff
@@ -0,0 +1,68 @@
+There are two principal mechanisms for formatting strings in C#/.NET. Use of `String.FormatO()` and string interpolation.
```

Comment:
There is quite a bit of text in this introduction, but I am hard pressed to find something that can be omitted. It is explained very well!